modules/api/src/main/java/org/eclipse/xpanse/api/controllers/IsvCloudCredentialsApi.java

```diff
@@ -148,6 +147,10 @@ public void deleteIsvCloudCredential(
             @Parameter(name = "name", description = "The name of credential.")
                     @RequestParam(name = "name")
                     String name) {
-        credentialCenter.deleteCredential(csp, siteName, type, name, null);
+        credentialCenter.deleteCredential(csp, siteName, type, name, getManagedIsv());
+    }
+
+    private String getManagedIsv() {
```

Comment:
i think the word 'managedIsv' does not give a clear meaning here. 

I see the method names in the UserHelper needs to be corrected. 
getCurrentUserManageIsv() -> getIsvManagedByCurrentUser()
getCurrentUserManageCsp() -> getCspManagedByCurrentUser()

And then use the similar namings at other places.